### ISSUE TITLE
iOS 10 compatibility?

### ISSUE THREAD

**Issue by The-SamminAter** [boo-dev/AirPort#4]
I can connect gen1 AirPods to iOS 10.3.3, would it be possible to make this tweak available for iOS 10(.3.x)? If not, what is preventing this tweak from working on iOS 10?

**Comment by kuhcaktyerdna** [comment]
Agreed! ios 10 support will be great because it will be terrible to update and lose jailbreak
